docs/config/index.md

```diff
@@ -333,6 +333,18 @@ export default defineConfig(async ({ command, mode }) => {
 
   See [here](/guide/env-and-mode#env-files) for more about environment files.
 
+### envVariblePrefix
```

Comment:
I'm not sure whether `envVariblePrefix` is a good name? I'm open to a better name.

Reply:
My preference would be to use `envPrefix`. We can leave the final call to be discussed with the rest of the team though this week.

Reply:
I don't have strong opinion here, both names are okay

Reply:
Thanks for the suggestions. I'm changing to `envPrefix ` for now and waiting for the final call.